Storage/SQL/FaviconsTable.swift

```diff
@@ -72,7 +72,7 @@ class FaviconsTable<T>: GenericTable<Favicon> {
         if (cursor.count != 1) {
             return nil
         }
-        return (cursor[0] as Favicon).id
+        return (cursor[0] as? Favicon)?.id
```

Comment:
But my reason for using the safer optional variants in some cases was this: if a function already allowed returning nil, then I'd rather us return nil than crash the browser if a favicon is malformed in the DB.